manifests/kustomize/env/aws/multi-user/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ./base
+  - ./cache-deployer-cert-manager
```

Comment:
One question: Any reason that you want to apply your change to multi-user mode only? It is fine by me the current way you organize it, but I am just curious if you want to support cert manager in both multi-user mode and KFP standalone mode.